osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-administration.xsd

```diff
@@ -0,0 +1,28 @@
+<!-- Copyright 2015 Smart Society Services B.V. Licensed under the Apache 
```

Comment:
is dit een nieuwe of zou deze in sm-configuration kunnen? misschien even afstemmen met Paul en Gerbert ?

Reply:
ja deze is nieuw idd, ik heb hem geisoleerd omdat het een apart geval is

Reply:
zal even met Gerbert en Paul overleggen